apps/meteor/client/hooks/useEndpointData.ts

```diff
@@ -24,10 +24,12 @@ export const useEndpointData = <TPath extends PathFor<'GET'>>(
 			.then(resolve)
 			.catch((error) => {
 				console.error(error);
-				dispatchToastMessage({
-					type: 'error',
-					message: error,
-				});
+				if (error && typeof error === 'string' && error !== 'undefined') {
```

Comment:
Error doesn't need to be string. It can be a normal error object. Maybe change to `(error !== undefined)`?.

Reply:
Tbh I'd prefer we just remove this toast completely already. It shouldn't have been here in the first place.